core/io/missing_resource.h

```diff
@@ -57,6 +57,8 @@ class MissingResource : public Resource {
 	void set_recording_properties(bool p_enable);
 	bool is_recording_properties() const;
 
+	virtual String get_save_class() const override;
```

Comment:
`get_save_class()` is called by the resources savers when saving the class type to the resource. If it's a missing resource, we need to emit the original class type here or otherwise they will write the type as "MissingResource"